### PR TITLE
Move certificate preparation after user creation

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: cfy_manager/components/manager/manager.py [hunk 1]
```diff
@@ -20,16 +20,22 @@
 
 from ..base_component import BaseComponent
 from ..service_names import MANAGER
-from ..components_constants import CONFIG
+from ..components_constants import CONFIG, SERVICES_TO_INSTALL
+from ..service_components import QUEUE_SERVICE
+from ..service_names import (
```
Comment (barucoh) [inline cfy_manager/components/manager/manager.py]: Don't need the ()

File: cfy_manager/components/manager/manager.py [hunk 2]
```diff
@@ -20,16 +20,22 @@
...
+    RABBITMQ,
+)
+from ...config import config
```
Comment (barucoh) [inline cfy_manager/components/manager/manager.py]: This should be after the `from ... import constants`

File: cfy_manager/components/manager/manager.py (hunk 1 above)
Comment (geokala) [reply cfy_manager/components/manager/manager.py]: Updated

File: cfy_manager/components/manager/manager.py (hunk 2 above)
Comment (geokala) [reply cfy_manager/components/manager/manager.py]: Updated

**Pull request merged**